### DISCUSSION TITLE
Build a snap package

### DISCUSSION THREAD

**Discussion by binyamin** [jpochyla/psst#309] [Ideas]
Currently, the GitHub workflow builds a debian package (`.deb`). Which is a great way to support Debian and Ubuntu. If the workflow built a snap, you could quickly support Arch, Fedora, Solus, openSUSE, and many other distros.

- See [the snapcraft checklist](https://snapcraft.io/docs/snapcraft-checklist) in the docs.
- I'll play around with building a snap for this project. No promises, though.

**Comment by jpochyla** [comment]
Hi @binyamin, once we start having releases (see the [Beta milestone](https://github.com/jpochyla/psst/milestone/1)), I'll be glad to include as many packaging flavors as possible.
